Manito/Manito/Screens/CreateRoom/Component/InputDateView.swift

```diff
@@ -19,16 +19,15 @@ class InputDateView: UIView {
         label.font = .font(.regular, ofSize: 18)
         return label
     }()
-    private let dateBackView: UIView = {
-        let view = UIView()
-        view.backgroundColor = .darkGray
-        view.makeBorderLayer(color: .subOrange)
+    private let dateBackView: CalendarView = {
```

Comment:
p5;

혹시 변수명에 Back이 붙은게 어떤 의미인가요 ??
```suggestion
    private let calendarView = CalendarView()
```
는 어떨까요오?? 
^_______^

Reply:
ㅋㅋㅋㅋㅋㅋㅋㅋㅋ 아이 그 전 코드에서 뒷부분만 싹 바꾸느라 요래 바꿨네용^^ 수정하겠습니당.

Reply:
[50ff8de](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/97/commits/50ff8ded080450e1ceff51c638baf8be8ebaefbb)